source/GameAction.cpp

```diff
@@ -416,3 +431,20 @@ GameAction GameAction::Instantiate(map<string, string> &subs, int jumps, int pay
 
 	return result;
 }
+
+
+
+void GameAction::AddOutfitObjective(map<string, string> &subs, const std::map<const Outfit *, int> &outfitObjective)
```

Comment:
A couple comments within this method would make it more clear what's going on, especially as half of what happens is with 'it.first' and 'it.second', which isn't clear without looking at the references to see what's being in the 'outfitObjective' parameter.

Most notably for the 3 things being done within the for-loop.